qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/resolvers/IfTruthyValueResolverTest.java

```diff
@@ -0,0 +1,75 @@
+/*******************************************************************************
+* Copyright (c) 2022 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.qute.resolvers;
+
+import static com.redhat.qute.QuteAssert.assertHover;
+import static com.redhat.qute.QuteAssert.c;
+import static com.redhat.qute.QuteAssert.r;
+import static com.redhat.qute.QuteAssert.testCompletionFor;
+import static com.redhat.qute.QuteAssert.testDiagnosticsFor;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for 'ifTruthy' value resolver.
+ *
+ * @see https://quarkus.io/guides/qute-reference#built-in-resolvers
+ *
+ */
+public class IfTruthyValueResolverTest {
+
+	@Test
+	public void diagnostics() {
+		String template = "{@org.acme.Item item}\r\n" + //
+				"{item.ifTruthy}";
+		testDiagnosticsFor(template);
+
+		template = "{@org.acme.Item item}\r\n" + //
+				" \r\n" + //
+				"{item.name.ifTruthy(item.name)}";
+		testDiagnosticsFor(template);
+	}
+
+	@Test
+	public void completion() throws Exception {
+		String template = "{@org.acme.Item item}\r\n" + //
```

Comment:
test for completion should be:

 * item.| (--> ifTruthy should be shown)
 * item.ifTruthy('abcd).  (--> property, method of Item should be shown in another words copy / paste the result like https://github.com/redhat-developer/quarkus-ls/blob/aff6277327d33e5152c9afb8fad30bba7fa4b9a4/qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/services/completions/QuteCompletionInExpressionWithForSectionTest.java#L89 by adjusting range)